### PR TITLE
dump php

### PR DESCRIPTION
**Pull request by saeideng**

### CODE REVIEW

**Comment by antograssiot** [comment]
Strange I tackled that some time ago in #420. I wonder when it was lost and if others changes were also.

File: src/Template/Pages/home.ctp [hunk 1]
```diff
@@ -73,10 +73,10 @@ $cakeDescription = 'CakePHP: the rapid development PHP framework';
     <div class="columns large-6">
         <h4>Environment</h4>
         <ul>
-        <?php if (version_compare(PHP_VERSION, '5.5.9', '>=')): ?>
+        <?php if (version_compare(PHP_VERSION, '5.6', '>=')): ?>
             <li class="bullet success">Your version of PHP is 5.5.9 or higher (detected <?= PHP_VERSION ?>).</li>
```
Comment (antograssiot) [inline src/Template/Pages/home.ctp]: Would you please also update this one.. ?

**Comment by saeideng** [comment]
> Strange I tackled that some time ago in #420. I wonder when it was lost and if others changes were also.

see https://github.com/cakephp/app/commit/afa31e23beaf13c22a9da08c4f5ac6d30d5de26b

**Comment by antograssiot** [comment]
Thank you

**Pull request merged**